benchmark_runner/common/prometheus/metrics-default.yaml

```diff
@@ -1,267 +1,10 @@
-# API server
-- query: histogram_quantile(0.99, sum(rate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb!~"WATCH", subresource!="log"}[2m])) by (verb,resource,subresource,instance,le)) > 0
-  metricName: API99thLatency
 
-- query: sum(irate(apiserver_request_total{apiserver="kube-apiserver",verb!="WATCH",subresource!="log"}[2m])) by (verb,instance,resource,code) > 0
-  metricName: APIRequestRate
+# container CPU/Memory
 
-- query: sum(apiserver_current_inflight_requests{}) by (request_kind) > 0
-  metricName: APIInflightRequests
+- query: (sum(irate(container_cpu_usage_seconds_total{container!="",namespace="benchmark-runner"}[2m])) by (node)) > 0
+  metricName: containerCPU-benchmark-runner
 
-# Container & pod metrics
-- query: (sum(container_memory_rss{name!="",container!="POD",namespace=~"openshift-(etcd|oauth-apiserver|.*apiserver|ovn-kubernetes|sdn|ingress|authentication|.*controller-manager|.*scheduler)"}) by (container, pod, namespace, node) and on (node) kube_node_role{role="master"}) > 0
-  metricName: containerMemory-Masters
```

Comment:
Agree, information about masters isn't going to be very interesting, but you should keep the comment.

Reply:
I run just 2 queries CPU and memory and we need more queries I will add it later